modules/imaging_uploader/static/schema.yml

```diff
@@ -0,0 +1,68 @@
+openapi: 3.0.1
```

Comment:
@cmadjar I think you might be the best person to review if this schema is accurate?